backend/learnings/views.py

```diff
@@ -65,3 +65,32 @@ def get(self, request, user):
         get_category = Category.objects.filter(~Q(quiztaken__user_id__username=user))
         serializer = CategorySerializer(get_category, many=True)
         return Response({"categories": serializer.data})
+
+
+class GetWordsPerCategory(APIView):
+    permission_classes = [IsAuthenticated]
+    authentication_classes = [TokenAuthentication]
+
+    def get(self, request, category_id):
+        word_per_category_with_choices = []
+        get_category = Category.objects.get(id=category_id)
+        get_words = Word.objects.filter(category_id=get_category)
+        for word in get_words:
+            get_choices = Choices.objects.get(word_id=word.id)
+            word_per_category_with_choices.append(
+                {
+                    "word_id": get_choices.id,
```

Comment:
I made a mistake in previous pr. This should've been included in pr https://github.com/framgia/sph-els-gift-canovas/pull/12 , but i approved and merged it already lets include this in this pr na lang :bow: